docs/basic-features/image-optimization.md

```diff
@@ -11,199 +11,171 @@ description: Next.js supports built-in image optimization, as well as third part
 
-## Image Component
+### Local Images
 
-To add an image to your application, import the [`next/image`](/docs/api-reference/next/image.md) component:
+To use a local image, `import` your `.jpg`, `.png`, or `.webp` files:
 
 ```jsx
+import profilePic from '../public/me.png'
+```
+
+Dynamic `await import()` or `require()` are _not_ supported.
+
+Next.js will automatically determine the `width` and `height` or your image based on the imported file. These values are used to prevent [Cumulative Layout Shift](https://nextjs.org/learn/seo/web-performance/cls) while your image is loading.
+
+```js
 import Image from 'next/image'
+import profilePic from '../public/me.png'
 
 function Home() {
   return (
     <>
       <h1>My Homepage</h1>
       <Image
-        src="/me.png"
+        src={profilePic}
         alt="Picture of the author"
-        width={500}
-        height={500}
+        // width={500} automatically provided
+        // height={500} automatically provided
+        // blurDataURL="data:..." automatically provided
+        // placeholder="blur" // Optional blur-up while loading
       />
       <p>Welcome to my homepage!</p>
     </>
   )
 }
-
-export default Home
 ```
 
-## Image Imports
-
-You can statically `import` images that live in your project. Dynamic `await import()` or `require()` are _not_ supported.
+### Remote Images
 
-With static `import`s, you only need to provide the `src` prop. The `width`, `height`, and `blurDataURL` props will automatically be populated. Alt text is still needed separately.
+To use a remote image, the `src` property should be a URL string, which can be [relative](#loaders) or [absolute](#domains). Because Next.js does not have access to remote files during the build proccess, you'll need to provide the [`width`](/docs/api-reference/next/image.md#width), [`height`](/docs/api-reference/next/image.md#height) and optional [`blurDataURL`](/docs/api-reference/next/image.md#blurdataurl) props manually:
 
-```js
+```jsx
 import Image from 'next/image'
-import profilePic from '../public/me.png'
 
-function Home() {
+export default function Home() {
   return (
     <>
       <h1>My Homepage</h1>
       <Image
-        src={profilePic}
+        src="/me.png"
         alt="Picture of the author"
-        // width={500} automatically provided
-        // height={500} automatically provided
-        // blurDataURL="data:..." automatically provided
-        // placeholder="blur" // Optional blur-up while loading
+        width={500}
+        height={500}
       />
       <p>Welcome to my homepage!</p>
     </>
   )
 }
 ```
 
-For remote images, you'll need to provide the [`width`](/docs/api-reference/next/image.md#width), [`height`](/docs/api-reference/next/image.md#height) and [`blurDataURL`](/docs/api-reference/next/image.md#blurdataurl) props manually.
+> Learn more about the [sizing requirements](#image-sizing) in `next/image`.
 
-## Properties
+### Loaders
```

Comment:
I think `domains` should be explained before `loader` because I think most people will use the default loader which would require `domains` configured for remote images.

We don't want to give users a foot-gun by telling them to use a `loader` first which could cause them to serve the image without resizing or reformatting it.

Reply:
Good point. Done!